PilotDataPlatform/helm-charts
# PILOT-5524: Remove command args from workspace service

## Summary

- Remove command and args from workspace service helm chart

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-5524

## Type of Change


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions

- Workspace service should deploy successfully without error that it cannot find `run.py`
